src/content/decentralized-identity.md

```diff
@@ -0,0 +1,157 @@
+
+## Decentralized identity use-cases {#decentralized-identity-use-cases}
+
+Decentralized identity has many potential use-cases:
+
+### 1. Universal dApp logins {#universal-dapp-logins} 
+[Decentralized applications](/dapps/) can issue attestations to users, which can be stored in an Ethereum wallet. An attestation in this case could be an an [NFT](/nft/) granting the holder access to an online community. 
+
+A [Sign-In With Ethereum](https://eips.ethereum.org/EIPS/eip-4361) function would then enable servers to confirm the user's Ethereum account and fetch the required attestation from their account address. This means users can access the dapp without inputting passwords or sharing personal information.  
+
+### 2. KYC Authentication {#kyc-authentication}
+Using many online services requires individuals to provide attestations and credentials, such as a driving license or national passport. But this approach is problematic because private user information can be compromised and service providers cannot verify the authenticity of the attestation. 
+
+Decentralized identity allows companies to skip on conventional Know-Your-Customer (KYC) processes and authenticate user identities via Verifiable Credentials. This reduces the cost of identity management and prevents the use of fake documentation. 
+
+### 3. Voting and online communities {#voting-and-online-communities}
+
+Online voting and social media are two novel applications for decentralized identity. Online voting schemes are susceptible to manipulation, especially if malicious actors create false identities to vote. Asking individuals to present on-chain attestations can improve the integrity of online voting processes. 
+
+Decentralized identity can enable the creation of online communities that are free of fake accounts. Each user must authenticate their identity using an on-chain identity system, like the Ethereum Name Service, reducing the possibility of bots overrunning social platforms. 
+
+## Use decentralized identity applications on Ethereum
+
+There are also many ambitious projects using Ethereum as a foundation for decentralized identity solutions:
+
+**Ethereum Name Service (ENS) - _A decentralized naming system for on-chain, machine-readable identifiers, like, Ethereum wallet addresses, content hashes, and metadata._**
+- [Website](https://ens.domains/)
+
+**SpruceID - _A decentralized identity project which allows users to control digital identity with Ethereum accounts and ENS profiles instead of relying on third-party services._** 
+- [Website](https://www.spruceid.com/)
+
+**Proof of Humanity - _Proof of Humanity (or PoH) is a social identity verification system built on Ethereum._**
+- [Website](https://www.proofofhumanity.id/)
+
+**BrightID - _A decentralized, open-source social identity network seeking to reform identity verification through the creation and analysis of a social graph._**
+- [Website](https://www.brightid.org/)
```

Comment:
Just updating as per site conventions:

```suggestion
- **[Ethereum Name Service (ENS)](https://ens.domains/)** - _A decentralized naming system for on-chain, machine-readable identifiers, like, Ethereum wallet addresses, content hashes, and metadata._
- **[SpruceID](https://www.spruceid.com/)** - _A decentralized identity project which allows users to control digital identity with Ethereum accounts and ENS profiles instead of relying on third-party services._*
- **[Proof of Humanity](https://www.proofofhumanity.id)** - _Proof of Humanity (or PoH) is a social identity verification system built on Ethereum._
- **[BrightID](https://www.brightid.org/)** - _A decentralized, open-source social identity network seeking to reform identity verification through the creation and analysis of a social graph._

```